config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'users/new'
+  post 'users/new', action: :create, controller: 'users'
```

Comment:
Instrad If naming It as 'users/new' name It as 'users/create'. It is a Better convention